### ISSUE TITLE
Streaming extern audio to pushtotalk.py

### ISSUE THREAD

**Issue by kenanEkici** [googlesamples/assistant-sdk-python#212]
Hi, I am currently using ROS in order to publish an external audio source on a remote host via a topic (over the network) to the Google Assistant. I was wondering how I could input the incoming stream of sound data to the pushtotalk module. I was thinking of using audio_helpers in pushtotalk.py but I honestly do not know how to even begin working with this raw sound data. Help would be highly appreciated.

**Comment by Fleker** [comment]
When you use the pushtotalk module, you'll get raw audio bytes from the network call. You can pipe these audio bytes to anywhere you want (the same for sending audio). Depending on what your audio system is, you can play the audio with whatever your remote host uses.

**Comment by kenanEkici** [comment]
I solved it differently. Basically, I used ROS' audio_common package to set up an audio capturing node on a remote host and another node on the client to play back the captured audio over a certain topic. Setting up the playback stream as loopback device with PulseAudio on the client enabled a Google Assistant client to listen to a remote (over the network) audio stream. This way I dont have to change any code or logic in the Google Assistant samples or helpers.
